plugins/qt3dinspector/geometryextension/buffermodel.cpp

```diff
@@ -83,16 +83,18 @@ void BufferModel::updateAttributes()
 
 void BufferModel::updateAttribute(const GammaRay::Qt3DGeometryAttributeData &attrData)
 {
-    m_rowSize = m_buffer.size() / attrData.count;
```

Comment:
Yea, I'm not sure. I only added the if statement around the original code. The if statement prevented a crash when loading buffers that had no data, I think.